test/shared.ts

```diff
@@ -37,3 +37,21 @@ export function truePath(...paths: string[]) {
     const basePath = path.join(__dirname, "..", "..", "test");
     return path.join(basePath, ...paths);
 }
+
+export async function setupTestEnv() {
+    const extension = await extensions.getExtension("redhat.java")!.activate();
```

Comment:
No need to activate here

Reply:
But the promise below will keep waiting until LS is ready ins standard mode. If we do not activate, are there chances that whole process will stuck?